modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/DynamicConfigExpirationTestCase.java

```diff
@@ -31,7 +35,7 @@
 
 public class DynamicConfigExpirationTestCase extends AbstractExtensionFunctionalTestCase {
 
-  private static final int PROBER_TIMEOUT = 20000;
+  private static final int PROBER_TIMEOUT = 30000;
```

Comment:
Can we make the tests run faster? it seems unnecessary for them to take more than 20 seconds.

Reply:
sure